src/RestQuery.js

```diff
@@ -97,6 +97,31 @@ function RestQuery(
     }
   }
 
+  if (Object.prototype.hasOwnProperty.call(restOptions, 'excludeKeys')) {
+    const keysForInclude = restOptions.excludeKeys
```

Comment:
Can you refactor this since its the same as `keys`?

Reply:
I'm not sure what you mean, like change it to `excludeKeysForInclude`? If so, I left it this way because of scope and  because it was still adding keys to `include`. I can change it if you think it will read better

Reply:
Actually this code looks identical to keys https://github.com/parse-community/parse-server/blob/0f5928542c44fb8b72ddca4a2d35c0905e2d335d/src/RestQuery.js#L75

This doesn't look right to me. Check the code coverage you are missing a test here

Reply:
It's similar, but not identical. The key's block https://github.com/parse-community/parse-server/blob/0f5928542c44fb8b72ddca4a2d35c0905e2d335d/src/RestQuery.js#L75-L98 

was already there and added in the comment above it.

The second one I added in this PR. Basically, if someone adds `excludeKeys`, the excludeKey nested fields should be added to `include`, the same way as `keys`

Reply:
The main difference is here: https://github.com/parse-community/parse-server/blob/0f5928542c44fb8b72ddca4a2d35c0905e2d335d/src/RestQuery.js#L100-L101

Reply:
I can probably combine the lists into one `if` block to simplify the code. I tried to stay away from changing as much of the original code as possible

Reply:
Yeah thats what I meant similar functionality. Can you write a test case to be sure?

https://app.codecov.io/gh/parse-community/parse-server/compare/7242/diff#D1L120